Assignment-1/services/src/parseinput.go

```
@@ -0,0 +1,61 @@
+package src
+
+import (
+	"bufio"
+	"fmt"
+	"nuclei-assignment-1/utils"
+	"os"
+	"strings"
+)
+
+func ParseInputName() string {
+	name := ""
+	reader := bufio.NewReader(os.Stdin)
+	InputNameDetails()
+	for name == "" {
+		text, _ := reader.ReadString('\n')
```

never ignore errors.

---

fixed it